mindsdb_native/libs/data_sources/gcs_ds.py

```diff
@@ -0,0 +1,39 @@
+import os 
+
+from google.cloud import storage
+
+from mindsdb_native.libs.data_types.data_source import DataSource
+from mindsdb_native.libs.data_sources.file_ds import FileDS
+from mindsdb_native import F
+
+
+class GCSDS(DataSource):
+
+    def _setup(self, bucket_name, file_path, path_to_auth_json):
```

Comment:
Isn't there some default path to the GC credentials like there is for aws ? If so, can you default `path_to_auth_json` to that path

Reply:
I haven't found any default credentials even after extensively researching so I am not sure how to go about it. The json file usually downloads in the default Download folder.